tensorflow/python/module/module_test.py

```diff
@@ -335,7 +335,7 @@ def testEntersNameScope_concreteFunction(self):
 class AbcTest(test_util.TensorFlowTestCase):
 
   def testAbstract(self):
-    msg = "Can't instantiate .* abstract methods"
+    msg = "Can't instantiate .* abstract method"
     with self.assertRaisesRegex(TypeError, msg):
```

Comment:
It would be ok to me to drop the regex altogether - knowing that a TypeError is raised is more than sufficient for this test.

Reply:
@mdanatg FYI - there's a couple instances of `assertRaisesRegex` in this test suite, I can remove them all if you'd like?

Reply:
Sounds good. Only those asserts that verify messaged produced by module.py itself should use regexps.

Reply:
@mdanatg seems like this particular test was the only assert that didn't need a regexp assert, so I went ahead and removed it for that test only.